### PR TITLE
Create make_grey_table.py

### PR DESCRIPTION
**Pull request by haakoan**
Python script used to create energy-integrated tables.

### CODE REVIEW

**Comment by evanoconnor** [comment]
maybe we put this in the main directory (rather than the src directory; but still in a script directory).

**Comment by haakoan** [comment]
Done

File: src/scripts/make_grey_table.py [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    """
+    # Read NuLib and EOS tables
+    nulib_table = read_nulib_table(nulib_file)
+    eos_table = read_eos_table(eos_file)
+
+    # Generate log-spaced neutrino temperature array
+    log_temperatures = np.logspace(np.log10(temperature_range[0]), np.log10(temperature_range[1]), num=num_temperatures)
+
+    # Initialize the 4D table array
+    table4d = np.zeros([3, 4, 82, 100, 56, len(log_temperatures)])
```
Comment (evanoconnor) [inline src/scripts/make_grey_table.py]: This hard coded for a particular table

Comment (haakoan) [reply src/scripts/make_grey_table.py]: Changed to account for the actual size of the original table. The two first dimensions have to be 3 and 4, the script will not work as is for more than 3 species.

**Review by evanoconnor — APPROVED** [review]
Thanks Haakon, will be useful for making grey tables from Nulib!

**Pull request merged**